src/Symfony/Bridge/Twig/CHANGELOG.md

```diff
@@ -5,8 +5,10 @@ CHANGELOG
 -----
 
  * added `AppVariable::getToken()`
- * Deprecated the possibility to inject the Form Twig Renderer into the form
-   extension. Inject it on TwigRendererEngine instead.
+ * [BC BREAK] Deprecated the possibility to inject the Form `TwigRenderer` into the form extension. 
```

Comment:
Well, the BC break is that you have to configure a runtime loader for the `TwigRenderer`.